openshift/release/create-release-branch.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Copyright 2019 The OpenShift Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: create-release-branch.sh v0.4.1 release-0.4
+
+set -ex # Exit immediately on error.
+
+source openshift/release/common.sh
+
+release=$1
+target=$2
+
+# Fetch the latest tags and checkout a new branch from the wanted tag.
+git fetch upstream -v --tags
+git checkout -b "$target" "$release"
+
+# Remove GH Action hooks from upstream
+rm -rf .github/workflows
+git commit -sm ":fire: remove unneeded workflows" .github/
+
+# Copy the midstream specific files from the OPENSHIFT/main branch.
+git fetch openshift main
+git checkout openshift/main -- "$MIDSTREAM_CUSTOM_FILES"
```

Comment:
Don't we want splicing here?

Reply:
I mean we probably do not want the `""` here.

Reply:
also: `error: pathspec 'OWNERS_ALIASES' did not match any file(s) known to git`

Reply:
> Don't we want splicing here?

:+1: updated it.

> also: error: pathspec 'OWNERS_ALIASES' did not match any file(s) known to git, does that file exist?

My fault. Checked on upstream... Updated it.